src/main/java/frc/robot/DriveSystem.java

```diff
@@ -5,56 +5,42 @@
 public class DriveSystem {
     private double _driveCmd;
     private double _turnCmd;
-
+    //Initialization of variables
     private Talon _motorFL;
     private Talon _motorFR;
     private Talon _motorRL;
     private Talon _motorRR;
 
-    /**
-     * Default constructor
-     * 
-     * Instantiates the motor Talon objects with their correct channels and inversion state based
-     * on constants contained in the RobotConstants class, e.g. -
-     * 
-     *      MOTOR_CHANNEL_FL, MOTOR_CHANNEL_FR, ...
-     *      MOTOR_INVERTED_FL, MOTOR_INVERTED_FR, ...
-     * 
-     * Also sets the commands to 0.0 by default.
-     */
+    //MOTOR_CHANNEL_FL, MOTOR_CHANNEL_FR, ...
+    //MOTOR_INVERTED_FL, MOTOR_INVERTED_FR, ...
     public DriveSystem() {
+        this._motorFL = new Talon(RobotConstants.MOTOR_CHANNEL_FL);
+        this._motorFR = new Talon(RobotConstants.MOTOR_CHANNEL_FR);
+        this._motorRL = new Talon(RobotConstants.MOTOR_INVERTED_FL);
+        this._motorRR = new Talon(RobotConstants.MOTOR_INVERTED_RR);
```

Comment:
Take a look at the WPILib documentation. There is a command for Talons to invert their direction.

Keep in mind that constructing a Talon to receive signals over a RoboRIO channel (as you've done here for FL and FR) is separate from configuring the direction the motor turns (which you haven't done)